### DISCUSSION TITLE
Transformers4Rec Tutorial: Triton Inference Server Failed to initialize Python stub + ModuleNotFoundError: No module named 'nvtabular', 'merlin'

### DISCUSSION THREAD

**Discussion by zwei2016** [NVIDIA-Merlin/Transformers4Rec#775] [Q&A]
Hi everyone 

I love the Transformers4Rec and try to learn through the online tutorial. I followed the 

> https://github.com/NVIDIA-Merlin/Transformers4Rec/tree/main/examples/getting-started-session-based

In the 3rd notebook : 03-serving-session-based-model-torch-backend.ipynb, I tried to launch the Triton Inference Server, but failed. 

The error message contains:   

> Failed to initialize Python stub + ModuleNotFoundError: No module named 'nvtabular', 'merlin'

I deployed Triton Inference Server through NGC container, as described in https://github.com/triton-inference-server/server, the Triton server returned the expected result as in "Serve a Model in 3 Easy Steps". I am not sure if I miss something when I followed the Transformers4Rec tutorial, could you please give me a clue?

By the way, I also report the details here:

> https://github.com/triton-inference-server/server/issues/7158

Thank you in advance 

Wei

**Comment by karinacarre** [comment]
We experience the same errors, Did you manage to get the library run correctly?
